PWGCF/Femto/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
-#add_subdirectory(DataModel)
```

Comment:
I guess this can be kept in the future because you already incorporated an empty `CMakeLists.txt` there